src/components/kepler-gl.js

```diff
@@ -279,6 +287,9 @@ function KeplerGlFactory(
               this.root = node;
             }}
           >
+            {uiState.notifications.length !== 0 ?
```

Comment:
even if `uiState.notifications.length === 0` `NotificationPanel` should still work right? I would profer apply less logic here and just pass `uiState.notifications` to `< NotificationPanel>` to let it figure out what to do.

Reply:
done